src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -82,55 +129,39 @@ class Comment extends React.Component<Props, State> {
         this.approvalCommentFormSubmitHandler();
     };
 
-    toEdit = (): void => this.setState({ isEditing: true, isInputOpen: true });
-
-    handleCommentFocus = (): void => {
-        this.setState({ isFocused: true });
-    };
-
-    handleCommentBlur = (): void => {
-        this.setState({ isFocused: false });
-    };
-
     render(): React.Node {
         const {
             avatarRenderer = identity,
             created_by,
             created_at,
-            permissions,
+            permissions = {},
```

Comment:
The default value here should probably live in `defaultProps`.

Reply:
agreed, we saw a mix of both in this component. when adding changes for the edit task menu, will pull these up to the `defaultProps`